blobfile/_azure.py

```diff
@@ -151,31 +151,24 @@ def _create_access_token_request(
     creds: Mapping[str, str], scope: str, success_codes: Sequence[int] = (200,)
 ) -> Request:
     if "refreshToken" in creds:
-        # https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-protocols-oauth-code#refreshing-the-access-tokens
+        # https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow#refresh-the-access-token
         data = {
             "grant_type": "refresh_token",
             "refresh_token": creds["refreshToken"],
-            "resource": scope,
+            "scope": scope,
         }
         tenant = "common"
     else:
-        # https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-oauth2-client-creds-grant-flow#request-an-access-token
-        # https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-protocols-oauth-code
-        # https://docs.microsoft.com/en-us/rest/api/storageservices/authorize-with-azure-active-directory#use-oauth-access-tokens-for-authentication
-        # https://docs.microsoft.com/en-us/rest/api/azure/
-        # https://docs.microsoft.com/en-us/rest/api/storageservices/authorize-with-azure-active-directory
-        # az ad sp create-for-rbac --name <name>
-        # az account list
-        # az role assignment create --role "Storage Blob Data Contributor" --assignee <appid> --scope "/subscriptions/<account id>"

```

Comment:
Can you update the commands here for creating an account that works with this method?

Reply:
Ah, sure - the steps would be identical. I'll add them back to the comment.

Reply:
Done!